app/src/main/java/com/tomtom/sdk/examples/usecase/MapModeSelectionActivity.kt

```diff
@@ -0,0 +1,145 @@
+package com.tomtom.sdk.examples.usecase
+
+import android.content.Intent
+import android.graphics.PorterDuff
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.Button
+import android.widget.ImageButton
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
+import com.google.android.material.card.MaterialCardView
+import com.tomtom.sdk.examples.R
+
+
+class MapModeSelectionActivity : AppCompatActivity() {
+    private var onlineButton: MaterialCardView? = null
+    private var offlineButton: MaterialCardView? = null
+    private var hybridButton: MaterialCardView? = null
+    private var mapMode = MapMode.ONLINE
+
+    companion object {
+        private const val STROKE_WIDTH = 4
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_map_mode_selection)
+        val toolbar = findViewById<View>(R.id.toolbar) as? Toolbar
+        setSupportActionBar(toolbar);
+
+        supportActionBar?.setDisplayHomeAsUpEnabled(true);
+
+        val onlineButtonId = R.id.online_button
+        val hybridButtonId = R.id.hybrid_button
+        val offlineButtonId = R.id.offline_button
```

Comment:
Why do assign these ids?
Have you considered using view binding to make it more concise? https://developer.android.com/topic/libraries/view-binding

Reply:
I couldn't find any usage of the binding feature in the project, and therefore, I didn't want to disrupt the integrity of the project. If i implement binding here, it should be added to everywhere, therefore I believe it should have been done in another PR.